src/cswinrt/strings/additions/Microsoft.UI.Xaml.Media.Media3D/Microsoft.UI.Xaml.Media.Media3D.Matrix3D.cs

```diff
@@ -418,7 +411,7 @@ private string ConvertToString(string format, IFormatProvider provider)
             }
 
             // Helper to get the numeric list separator for a given culture.
-            char separator = TokenizerHelper.GetNumericListSeparator(provider);
+            char separator = global::ABI.Windows.Foundation.TokenizerHelper.GetNumericListSeparator(provider);
             return string.Format(provider,
```

Comment:
Use `DefaultInterpolatedStringHandler` here too. Basically everywhere we do `string.Format` and similar.